client/ayon_ftrack/plugins/publish/integrate_hierarchy_ftrack.py

```diff
@@ -273,7 +278,11 @@ def import_to_ftrack(
 
             # Create entity if not exists
             if entity is None:
-                folder_type = entity_data["folder_type"]
+                # Sanitize against case sensitive folder types.
+                folder_type = object_types_by_lower_name[
+                    entity_data["folder_type"].lower()
+                ]["name"]
```

Comment:
```suggestion
                folder_type_low = entity_data["folder_type"].lower()
                object_type = object_types_by_lower_name[folder_type_low]
                entity_type = object_type["name"].replace(" ", "")
```
Requires to change 286 to
`entity = session.create(entity_type, {`